osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/infra/jms/messageprocessor/DomainResponseMessageProcessorTest.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.ws.smartmetering.infra.jms.messageprocessor;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+
+import javax.jms.JMSException;
+import javax.jms.ObjectMessage;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opensmartgridplatform.adapter.ws.shared.services.NotificationService;
+import org.opensmartgridplatform.adapter.ws.shared.services.ResponseDataService;
+
+@ExtendWith(MockitoExtension.class)
+public class DomainResponseMessageProcessorTest {
+
+    @Mock
+    private NotificationService notificationService;
+
+    @Mock
+    private ResponseDataService responseDataService;
+
+    @Mock
+    private ObjectMessage message;
+
+    @InjectMocks
+    private BundleResponseMessageProcessor messageProcessor;
+
+    @BeforeEach
+    void init() throws JMSException {
+        when(this.message.getJMSType()).thenReturn("HANDLE_BUNDLED_ACTIONS");
+        when(this.message.getStringProperty("Result")).thenReturn("OK");
+        when(this.message.getStringProperty("OrganisationIdentification")).thenReturn("some-organisation");
+        when(this.message.getJMSCorrelationID()).thenReturn(null);
+        when(this.message.getStringProperty("DeviceIdentification")).thenReturn(null);
+        when(this.message.getStringProperty("Description")).thenReturn(null);
+        when(this.message.getObject()).thenReturn(null);
+    }
+
+    @Test
+    void processMessageSuccessfully() throws JMSException {
+        this.messageProcessor.processMessage(this.message);
+
+        verify(this.responseDataService).enqueue(any());
+        verify(this.notificationService).sendNotification(any(), any());
+    }
+
+    @Test
+    void processMessageErrorInHandleMessage() throws JMSException {
+
+        org.mockito.Mockito.doThrow(new NullPointerException("Some runtime exception"))
```

Comment:
The fully qualified  `org.mockito.Mockito.doThrow` looks a bit out of tone with the other Mockito methods with static imports. When using methods from a single place (`org.mockito.Mockito`) I think it is cleaner to be consistent,
Given how common these are in test code, I'd say using static imports for all Mockito methods would be the way to go.

Reply:
Added org.mockito.Mockito.doThrow as a static import.